### ISSUE TITLE
DESeq2 error without choosing factors

### ISSUE THREAD

**Issue by gexijin** [iDEP-SDSU/idep#117]
Demo data --> DEG1
Select factors & comparisons
Skip the factor selection "1. Select 6 or less main factors. Or skip this step...." 
Select comparisons among sample groups.  "2. ...."
Error

**Comment by c2b2pss** [comment]
Yes. Further, selecting just 2 comparisons gives a DESeq2 error.

Has the issue been fixed? Is idep 0.92 usable for DE analysis?

Thanks anyways for the great effort!
